Source/MediaStorageAndFileFormat/gdcmAnonymizer.cxx

```diff
@@ -1116,7 +1120,14 @@ bool Anonymizer::BALCPProtect(DataSet &ds, Tag const & tag, IOD const & iod)
         {
         if( const ByteValue *bv = copy.GetByteValue() )
           {
-          UIDToAnonymize = std::string( bv->GetPointer(), bv->GetLength() );
+          // UI VR with odd length get padded with a null byte
```

Comment:
We should probably chat about this comment.

Reply:
Thanks, I was not totally sure if this was correct